migrations/11_cron.up.sql

```diff
@@ -1,8 +1,9 @@
 CREATE EXTENSION IF NOT EXISTS pg_cron;
 
 SELECT cron.schedule('remove-old-verifications', '0 0 * * *', $$DELETE FROM verifications WHERE created_at < CURRENT_TIMESTAMP - interval '1 hours'$$);
-SELECT cron.schedule('remove-old-ake-states', '0 0 * * *', $$DELETE FROM ake_states WHERE created_at < CURRENT_TIMESTAMP - interval '1 hours'$$);
-SELECT cron.schedule('remove-old-registration-states', '0 0 * * *', $$DELETE FROM registration_states WHERE created_at < CURRENT_TIMESTAMP - interval '1 hours'$$);
+SELECT cron.schedule('remove-old-interim-password-states', '0 0 * * *', $$DELETE FROM interim_password_states WHERE created_at < CURRENT_TIMESTAMP - interval '1 hours'$$);
 SELECT cron.schedule('clean-job-history', '0 0 * * *', $$DELETE 
     FROM cron.job_run_details 
     WHERE end_time < now() - interval '7 days'$$);
+
+SELECT cron.schedule('remove-old-totp-used-codes', '0 0 * * *', $$DELETE FROM totp_used_codes WHERE created_at < CURRENT_TIMESTAMP - interval '1 hours'$$);
```

Comment:
If I'm reading this correctly (looking at `man 5 crontab`), `0 0 * * *` means that we'll only run this once a day, right? That's fine, but it does mean we'll hold on to these only-valid-for-1-hour records for much longer than needed.

Reply:
Yes! this is cleanup every day, but valid for an hour. We can do:

```
SELECT cron.schedule(
  'remove-old-totp-used-codes',
  '*/5 * * * *',  -- every 5 minutes
  $$DELETE FROM totp_used_codes
    WHERE created_at < CURRENT_TIMESTAMP - interval '1 hours'$$
);
```

but maybe too much.